src/translations/zh-Hans.yml

```diff
@@ -0,0 +1,1033 @@
+      t: 发展方向
+    - key: block.description.js_moving_in_right_direction
+      t: “JavaScript 正朝好的方向发展”
+
+    - key: block.title.building_js_apps_overly_complex
+      t: 复杂度
+    - key: block.description.building_js_apps_overly_complex
+      t: “构建 JavaScript 应用现在复杂过头了”
+
+    - key: block.title.js_over_used_online
+      t: 使用情况
+    - key: block.description.js_over_used_online
+      t: “JavaScript 目前在线上被过度使用了”
+
+    - key: block.title.enjoy_building_js_apps
+      t: 享受程度
+    - key: block.description.enjoy_building_js_apps
+      t: “我乐于创造 JavaScript 应用”
+
+    - key: block.title.would_like_js_to_be_main_lang
+      t: 主要语言
+    - key: block.description.would_like_js_to_be_main_lang
+      t: “我想把 JavaScript 作为我的主要编程语言”
+
+    - key: block.title.js_ecosystem_changing_to_fast
+      t: 变化频率
+    - key: block.description.js_ecosystem_changing_to_fast
+      t: “JavaScript 生态圈变得太快了”
+
+    - key: block.title.missing-features
+      t: 缺少的特性
+    - key: block.description.missing-features
+      t: 开发者们认为 JavaScript 目前缺少什么？
+
+      # opinion scale
+    - key: opinions.0.short
+      t: 强烈反对
+    - key: opinions.1.short
+      t: 反对
+    - key: opinions.2.short
+      t: 中立
+    - key: opinions.3.short
+      t: 同意
+    - key: opinions.4.short
+      t: 十分同意
+
+    - key: opinions.0.long
+      t: 强烈反对
+    - key: opinions.1.long
+      t: 反对
+    - key: opinions.2.long
+      t: 中立
+    - key: opinions.3.long
+      t: 同意
+    - key: opinions.4.long
+      t: 十分同意
+
+    ###########################################################################
+    # Awards
+    ###########################################################################
+
+    - key: awards.runner_ups
+      t: 亚军
+
+    # feature_adoption
+    - key: award.feature_adoption_award.title
+      t: 最受欢迎的特性
+    - key: award.feature_adoption_award.description
+      t: |
+          授予最受欢迎的 JavasScript 特性。
+    - key: award.feature_adoption_award.comment
+      t: |
+          ${items[0].name} 有 **${items[0].value}** 的用户使用，
+          是今年最常被使用的 JavaScript 特性。
+
+    # tool_satisfaction
+    - key: award.tool_satisfaction_award.title
+      t: 最高的满意度
+    - key: award.tool_satisfaction_award.description
+      t: |
+          授予拥有最高满意度的技术。
+
+    - key: award.tool_satisfaction_award.comment
+      t: |
+          有 **${items[0].value}** 的用户愿意再次使用它，
+          ${items[0].name} 再次证明了它可不是在开玩笑。
+
+    # tool_interest
+    - key: award.tool_interest_award.title
+      t: 最高关注度
+    - key: award.tool_interest_award.description
+      t: |
+          授予开发者最想学习的技术。
+
+    - key: award.tool_interest_award.comment
+      t: |
+          **${items[0].value}** 的开发者听说过 ${items[0].name} 并且想学习它，
+          那可是算是相当的关注！
+
+
+    # tool_usage
+    - key: award.tool_usage_award.title
+      t: 最常使用的技术
+    - key: award.tool_usage_award.description
+      t: |
+          授予拥有最大用户群的技术。
+    - key: award.tool_usage_award.comment
+      t: |
+          ${items[0].name} 有 **${items[0].value}** 的用户使用，
```

Comment:
@seognil Yes, I'd consider it makes more sense with "受访者" since the data is within the group of survey respondents.

@SachaG Based on the discussion above (where `用户` means `users` while `受访者` means `respondents` or `survey respondents`), would you consider it better to use the word `respondents` instead of `users`? To me, `users` is rather generic. While `respondents` indicate that the results are based on the survey respondents, which makes it more precise.
e.g. [Here](https://github.com/StateOfJS/State-of-JS-2019/blob/master/src/translations/en-US.yml#L596) it mentions `users who participated`, which is IMO totally fine. But for [here](https://github.com/StateOfJS/State-of-JS-2019/blob/master/src/translations/en-US.yml#L781), [here](https://github.com/StateOfJS/State-of-JS-2019/blob/master/src/translations/en-US.yml#L793), [here](https://github.com/StateOfJS/State-of-JS-2019/blob/master/src/translations/en-US.yml#L828), etc., what's your opinion, please?

Reply:
@SachaG @S1ngS1ng
I've got an idea
How about to make a description at top:

> （请注意：下文中的“用户”是指“受访用户”）
> (notice: 'users' below means 'survey respondents')

seems saving both meaning and readability

Reply:
@seognil You are a genius! 👏👏👏